redisenterprise/datadog_checks/redisenterprise/check.py

```diff
@@ -0,0 +1,309 @@
+import sys
+from datetime import datetime, timedelta
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+# from typing import Any
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'redisenterprise'
+
+
+class RedisenterpriseCheck(AgentCheck):
+    """RedisenterpriseCheck attempts to connect to the cluster and ensure the node is the master node"""
+
+    def __init__(self, name, init_config, instances):
+        super(RedisenterpriseCheck, self).__init__(name, init_config, instances)
+        # Set this to two minutes ago which may cause duplicates but we need to get everything in the case of failover
+        self.last_event_timestamp_seen = datetime.utcnow() - timedelta(0, 120)
+
+    def _timestamp(self, date):
+        """ Allows us to return an epoch time stamp if we use python2 or python3 """
+        if sys.version_info[0] < 3 or sys.version_info[1] < 4:
+            import time
+
+            return int(time.mktime(date.timetuple()))
+        else:
+            return int(date.timestamp())
+
+    def check(self, instance):
+        host = self.instance.get('host')
+        port = self.instance.get('port', 9443)
+        event_limit = self.instance.get('event_limit', 100)
+        is_mock = self.instance.get('is_mock', False)
+        service_check_tags = self.instance.get('tags', [])
+
+        if not host or not user or not password:
+            raise ConfigurationError('Configuration error, please fix redisenterprise.yaml')
+
+        try:
+
+            # noop if we are not the cluser master
+            if self._check_follower(host, port, is_mock):
+                self.last_timestamp_seen = datetime.utcnow()
+                pass
+
+            # add the cluster FQDN to the tags
+            fqdn = self._get_fqdn(host, port, service_check_tags)
+            service_check_tags.append('cluster:{}'.format(fqdn))
```

Comment:
```suggestion
            service_check_tags.append('redis_cluster:{}'.format(fqdn))
```
Cluster is a very generic tag that is used, for example, by the cluster agent. This can cause confusion and some metrics to either be incorrectly tagged or appear toghether when they shouldnt. Using a less generic name will help avoid those issues

Reply:
@hithwen I've modified the tag and all of the dashboards to reflect this change